config/locales/de.yml

```diff
@@ -1010,6 +1010,13 @@ de:
     events_in_other_conferences: Events in anderen Konferenzen
     feedback_not_enabled: Das Feedback-System ist nicht aktiviert.
     hide_all: Alles verbergen
+    indicator:
+      full:
+        submitted_after_hard_deadline: Eingereicht nach harter Frist
+        submitted_after_soft_deadline: Eingereicht nach Fristablauf
```

Comment:
from config/locales/de.yml, in German that would be "end_date: Enddatum
        hard_deadline: Harte Einreichungsfrist"

And I guess, these translations break conventions in other languages, as well.

Reply:
Your feedback is not clear enough for me. The "indicator full" is the string going for the event page, and the "indicator short" is the text going to the list pages. I thought just writing "Late" is enough becasue I did not want it to be too verbose

Reply:
Sure, the short version is okay. But the long version of the term "hard deadline" and "soft deadline" already has had other translations. And now these mismatch. I showed you with the German example. Other translations don't even show differences between the two translations.

Reply:
So its

submitted_after_soft_deadline: Eingereicht nach Enddatum
submitted_after_hard_deadline: Eingereicht nach Harte Einreichungsfrist

?